client/app/scripts/superdesk/filters.js

```diff
@@ -221,5 +221,12 @@
                 var conditionValue = labels.length > 0 ? labels.join(', ') : filterCondition.value;
                 return '(' + filterCondition.field + ' ' + filterCondition.operator + ' ' + conditionValue + ')';
             };
+        })
+        .filter('formatString', function() {
```

Comment:
Hmmm, maybe `removeLodash` is more appropriate? I'm such a bed author of function and class names :D